Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
+RUN apt-get update
```

Comment:
It's a best practice to run `apt-get update` and `apt-get install` in the same RUN command so that if you add another package to install, Docker reruns the update instead of using the old cache.

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run